assets/schema_input.json

```diff
@@ -23,6 +23,11 @@
                 "errorMessage": "Species of the reference",
                 "meta": ["species"]
             },
+            "bed_intervals": {
+                "type": "string",
+                "pattern": "^\\S+\\.bed?$",
+                "errorMessage": "TODO"
```

Comment:
I think you need the `errorMessage` here.
Also can a bed file really be `.be` ?

Reply:
yeah, I need error messages everywhere, I'll do this later.
`.be`? it's `.bed` here, no?

Reply:
If you use `.bed?` then the `d` is optional in the regex, normally.
You can try easily with https://regexr.com/

Reply:
:facepalm: of course